src/app/3d/school3d/ModelController/modelController.ts

```diff
@@ -9,7 +9,6 @@ import {
 } from 'three';
 import {OrbitControls} from 'three/examples/jsm/controls/OrbitControls';
 import {MqttInterface} from '../mqttInterface';
-import {delay} from 'rxjs/operators';
```

Comment:
I am not familiar with the rxjs delay method. Is there any reason you are implementing this yourself

Reply:
It wasn't working, so i replaced it

Reply:
ok, it should be fine here maybe look up what's not working with the rxjs version but that has low priority. rxjs version would still be preferred